docs/semgrep-cloud-platform/tags.md

```diff
@@ -23,13 +23,14 @@ import MoreHelp from "/src/components/MoreHelp"
 
 Add tags for specific projects in the Semgrep Cloud Platform through the following methods:
 
+* Set tags through the Semgrep Cloud Platform > Project page.
+* Set tags using the Semgrep Cloud Platform API (for Team and Enterprise Tier users).
 * Set tags in your repository's `.semgrepconfig.yml` file.
-* Set tags through Semgrep Cloud Platform > Project page or Semgrep Cloud Platform API (for Team and Enterprise Tier users).
 
-:::tip Best practices
-*  Choose either Semgrep Cloud Platform (and API) or `.semgrepconfig.yml` to manage tags. **Do not use a mix of the two.**
-* Semgrep Cloud Platform and its API **never** overwrite your `.semgrepconfig.yml` file. If you choose to use `.semgrepconfig.yml` to manage your tags, use it **exclusively**. Do not use Semgrep Cloud Platform or the API to manage any tags. Any changes to your tags through Semgrep Cloud Platform will be overwritten after a CI scan due to values present in `.semgrepconfig.yml`.
-* Semgrep always prioritizes values in `.semgrepconfig.yml` as the source of truth.
+:::caution Setting tags
+* Keep in mind, when setting tags via the `.semgrepconfig.yml` file or Semgrep Cloud Platform API, that these actions **overwrite** any tags previously set.
```

Comment:
I would want to be careful of any language that implies we may modify their physical configuration files, if present. Can we keep the existing comment that semgrep will never touch it or modify your changes to include some language around not modifying the files?

Reply:
i originally found the comment of us never overwriting the `.semgrepconfig.yml` file a bit confusing. it's a true statement but i was thinking why is this mentioned. what if we just change overwrite to supersede?

Reply:
```suggestion
* Keep in mind, when setting tags via the `.semgrepconfig.yml` file or Semgrep Cloud Platform API, that these actions **supersede** any tags previously set.
```

Reply:
I can understand that. I think it can be a difficult balancing game, if I don't mention it it could cause confusion, but if I do mention it that may raise more questions. 

I'm happy approving with the language you currently have :)

Reply:
May I suggest that Sina's primary improvement:
> Keep in mind, when setting tags via the `.semgrepconfig.yml` file or Semgrep Cloud Platform API, that these actions **overwrite** any tags previously set.

**Replace** this sentence:
> Any changes to your tags through Semgrep Cloud Platform will be overwritten after a CI scan due to values present in `.semgrepconfig.yml`.

All the other sentences are best practices, as the title of the callout states. If we only describe the way a feature behaves, we are missing out on guiding the user **on how we want them to use it / safest way for the user to use it to get best results**. If we want to reword it somehow, I'm open to that. 👍

Reply:
Your latest commit sounds good to me :) approving.